app/lib/fsHelper.js

```diff
@@ -18,7 +28,7 @@ function saveJsonSync(obj, filename) {
 
 function loadJsonSync(filename) {
   const path = `${OUTPUT_DIR}/${filename}.json`;
-  const jsonString = fs.existsSync(path) ? fs.readFileSync(path, 'utf8') : undefined;
+  const jsonString = fileExists(path) ? fs.readFileSync(path, 'utf8') : undefined;
```

Comment:
My original PR comment about existsSync is incorrect. The async version is the one which is deprecated.

The approach of 'checking existence' Vs 'do something and catch errors' is probably worth a slack discussion thread.